galleries/examples/images_contours_and_fields/image_exact_placement.py

```diff
@@ -0,0 +1,131 @@
+"""
+=========================================
+Placing images, preserving relative sizes
+=========================================
+
+By default Matplotlib resamples images created with `~.Axes.imshow` to
+fit inside the parent `~.Axes`.  This can mean that images that have very
+different original sizes can end up appearing similar in size.
+
+Sometimes, however,  it is desirable to keep the images the same relative size, or
+even to make the images keep exactly the same pixels as the original data.
+Matplotlib does not automatically make either of these things happen,
+but it is possible with some manual manipulation.
+
+Preserving relative sizes
+=========================
+
+If the size of the images are amenable, we can preserve the relative sizes of two
+images by using either the *width_ratio* or *height_ratio* of the subplots.  Which
+one you use depends on the shape of the image and the size of the figure.
```

Comment:
I suggest to move this below the "default layout" plot (to line 58).

Train of thought:
- Problem statement
- Example illustration of the problem
- General solution
- Specific solution example